Scripts/Connections.py

```diff
@@ -8,11 +8,22 @@
 
 redisPort = 6379
 stockPort = 4444
+transPort = 6666
 
 dbPort = 5432
 dbName, dbUser, dbPassword = "TBD"
 
 
+# Creates socket for Sending/Receiving from WebService
+def connectWeb():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.bind((localHost, transPort))
+    s.listen()
+    conn, addr = s.accept()
+    print("Connected to WebService @ " + addr[0])
```

Comment:
maybe just add the port as well so we can be double sure the connection is as we expect

Reply:
I can add most of them to the gitignore, the only one really need is redis-server. I don't know if it relies on anything.